server/core/templates/oauth2_access_denied.html

```diff
@@ -0,0 +1,16 @@
+(% extends "base.html" %)
+
+(% block title %)Oauth2 Access Denied(% endblock %)
+
+(% block head %)
+(% endblock %)
+
+(% block body %)
+	<h2>OAuth2 Access Denied</h2>
+	<main id="main">
+		<p>Access to this service was denied as you do not have the requisite scopes.</p>
```

Comment:
Requisite scopes etc isn't helpful for the user, just "denied" is enough in context.

```suggestion
```